src/acquisition/covidcast/database.py

```diff
@@ -251,41 +339,6 @@ def update_timeseries_timestamp2(
     args = (source, signal, time_type, geo_type, geo_value)
     self._cursor.execute(sql, args)
 
-  def get_covidcast_meta(self):
```

Comment:
Why was this removed?

Reply:
whoops!  looks like it was from a merge conflict...  i dont know how i missed that